stubs/tokenize.pyi

```diff
@@ -90,7 +90,14 @@ class TokenInfo(Tuple[int, str, Tuple[int, int], Tuple[int, int], int]):
     start: Tuple[int, int] = ...
     end: Tuple[int, int] = ...
     line: int = ...
-    def __repr__(self) -> str: ...
 
-def detect_encoding(readline: Callable[[], bytes]) -> Tuple[str, Sequence[bytes]]: ...
-def tokenize(readline: Callable[[], bytes]) -> Generator[TokenInfo, None, None]: ...
+    def __repr__(self) -> str:
+        ...
+
+
+def detect_encoding(readline: Callable[[], bytes]) -> Tuple[str, Sequence[bytes]]:
+    ...
+
+
+def tokenize(readline: Callable[[], bytes]) -> Generator[TokenInfo, None, None]:
+    ...
```

Comment:
Looks like this could/should be reported as a bug on µfmt for not detecting/telling black that it's formatting a type stub.